molecule_docker/playbooks/create.yml

```diff
@@ -67,7 +67,7 @@
         - not item.item.pre_build_image | default(false)
       docker_image:
         build:
-          path: "{{ molecule_ephemeral_directory }}"
+          path: "{{ item.item.context | default(molecule_ephemeral_directory) }}"
```

Comment:
Why would this be called context and not path? I also do not find this documented anywhere, which worries me a little bit.

Reply:
This is how docker is calling it. Because the image is built by the docker engine (that might not even run on the same host as the CLI command), this is what happens when we execute the build command (a bit simplified, but close enough):

 1. CLI tool packages the current directory into a build context and sends it to the engine.
 2. Engine executes instructions from the Dockerfile, taking any files required from the context that has been sent to it.

I hope this explains things a bit.